src/main/scala/scalapb/json4s/JsonFormat.scala

```diff
@@ -370,7 +370,7 @@ class Printer private (config: Printer.PrinterConfig) {
   def serializeSingleValue(
       fd: FieldDescriptor,
       value: PValue,
-      formattingLongAsNumber: Boolean
+      formattingLongAsNumber: Boolean = config.isFormattingLongAsNumber
```

Comment:
For binary compatibility with earlier versions, please add a new two parameter `(fd, value)` overload of this method that calls the three-param version with `config.isFormattingLongAsNumber`. Then, please change all calls to this method in this project to to the two parameter version.